src/components/pages/InstructorHome/index.js

```diff
@@ -1,23 +1,59 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
 import '../../../styles/InstructorStyles/index.less';
 import InstructorSidebar from './InstructorSidebar';
-import CalendarApp from '../../common/calendar/Calendar';
 import { Layout } from 'antd';
+import CourseCard from '../AdminHome/AdminCourseCard';
+import { useDispatch } from 'react-redux';
+import { getCourses } from '../../../redux/actions/instructorActions';
+import { useOktaAuth } from '@okta/okta-react';
+import { connect } from 'react-redux';
+//import AdminAddCoursesForm from '../AdminHome/AdminAddCoursesForm.js'
+//The above import will be used to add the add courses form, currently being developed in ticket BL-868
 
 const { Content } = Layout;
-const InstructorHome = () => {
+const InstructorHome = props => {
+  const { courses } = props;
+  const history = useHistory();
+  const dispatch = useDispatch();
+  const idToken = useOktaAuth().oktaAuth.getIdToken();
+
+  useEffect(() => {
+    dispatch(getCourses(idToken));
+  }, [dispatch, idToken]);
+
+  const handleClick = () => {
+    history.push('/admin-add-course');
+  };
+
   return (
     <div>
       <Layout>
         <InstructorSidebar />
         <Content>
-          <div className="calendar" data-testid="calendar">
-            <CalendarApp />
+          <div className="table-container">
+            <div className="courses-page-flex">
+              <div className="courses-right">
+                <button className="add-course" onClick={handleClick}>
+                  ADD COURSE
+                </button>
+              </div>
+            </div>
+            <div className="admin-courses-container">
+              {courses.map(course => (
+                <CourseCard key={course.course_id} course={course} />
+              ))}
+            </div>
           </div>
         </Content>
       </Layout>
     </div>
   );
 };
 
-export default InstructorHome;
+const mapStateToProps = state => {
+  return {
```

Comment:
This could also be more concise by leaving out the return line and closing bracket:
```
const mapStateToProps = state => {
    courses: state.instructorReducer.courses,
};
```

This will return the object directly because the fat arrow `=>` implies something being returned